docs/azure_ad_config_guide.rst

```diff
@@ -7,8 +7,8 @@ itself, it might take some tries to get all settings right.
 This guide tries to give a basic overview of how to configure Azure AD and how to determine the settings for
 django-auth-adfs. Installing and configuring the basics of Azure AD is not explained here.
 
-
-Step 1 - Register an application
+.. _step-1:
```

Comment:
Needs to be a newline between `.. _step-1` and the next block.
The `-----`s under the headlines must be the same length as the headline. I missed a few last PR too, so if you'd like to fix them all that would be awesome :beers:

```rst
.. _step-1:

Step 1 - Register a backend application
---------------------------------------
```